projects/packages/forms/src/blocks/contact-form/class-contact-form-block.php

```diff
@@ -295,21 +295,19 @@ public static function load_editor_scripts() {
 		$contact_form            = new Contact_Form( array() );
 		$defaults                = $contact_form->defaults;
 		$form_responses_url      = $dashboard_view_switch->get_forms_admin_url();
-		$form_responses_spam_url = $dashboard_view_switch->get_forms_admin_url( 'spam' );
 		$akismet_active_with_key = Jetpack::is_akismet_active();
 		$akismet_key_url         = admin_url( 'admin.php?page=akismet-key-config' );
 		$preferred_view          = $dashboard_view_switch->get_preferred_view();
 
 		$data = array(
 			'defaults' => array(
-				'to'                    => $defaults['to'],
-				'subject'               => $defaults['subject'],
-				'formsResponsesUrl'     => $form_responses_url,
-				'formsResponsesSpamUrl' => $form_responses_spam_url,
```

Comment:
Just this line is removed. It shows other lines changing, but it's just spacing.